test/onigumo_cli_test.exs

```diff
@@ -1,6 +1,7 @@
 defmodule OnigumoCLITest do
   use ExUnit.Case
   import Mox
+  import ExUnit.CaptureIO
```

Comment:
Can we group _ExUnit.Case_ and _ExUnit.CaptureIO_? Not only it belongs together, but _ExUnit_ comes alphabetically before _Mox_.